### ISSUE TITLE
XFP rules UI doesn't meet WCAG 2.1 contrast requirements

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#986]
**Describe the bug**
The text for the row labels have insufficient colour contrast to meet WGAC 2.1 at AA level.

![Screenshot 2019-06-03 at 13 38 08](https://user-images.githubusercontent.com/18653/58802340-d7cc6a80-8604-11e9-9004-b84b765d008e.png)
